pilot/cmd/pilot-agent/status/server_test.go

```diff
@@ -217,9 +217,10 @@ my_other_metric{} 0
 			if rec.Code != 200 {
 				t.Fatalf("handleStats() => %v; want 200", rec.Code)
 			}
-			if rec.Body.String() != tt.output {
+			if !strings.Contains(rec.Body.String(), tt.output) {
```

Comment:
Any reason this is not an exact match anymore?

Reply:
with the go metrics, you cant really predict them in the test i think. that was one of the issues i had, ill look into it.

Reply:
Oh we are adding those, makes sense. Thanks